code/modules/mob/living/carbon/alien/humanoid/humanoid_defense.dm

```diff
@@ -1,16 +1,25 @@
+/mob/living/carbon/alien/humanoid/attack_hulk(mob/living/carbon/human/user, does_attack_animation = FALSE)
```

Comment:
Does this need an attack log?

Reply:
Discussed this; `atom/proc/attack_hulk` should handle this properly.